google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -787,7 +787,8 @@ def _extract_cell(args, cell_body):
                          csv_delimiter=args['delimiter'], csv_header=args['header'],
                          compress=args['compress'])
   elif args['query'] or args['view']:
-    source = google.datalab.utils.commands.get_notebook_item(args['view'])
+    source_name = args['view'] if args['view'] else args['query']
```

Comment:
This feels redundant to me.

What about using `source_name = args['view'] or args['query']` instead?